scene/gui/control.cpp

```diff
@@ -3179,7 +3179,7 @@ void Control::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "clip_contents"), "set_clip_contents", "is_clipping_contents");
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "custom_minimum_size", PROPERTY_HINT_NONE, "suffix:px"), "set_custom_minimum_size", "get_custom_minimum_size");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "layout_direction", PROPERTY_HINT_ENUM, "Inherited,Locale,Left-to-Right,Right-to-Left"), "set_layout_direction", "get_layout_direction");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "layout_mode", PROPERTY_HINT_ENUM, "Position,Anchors,Container,Uncontrolled", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_INTERNAL), "_set_layout_mode", "_get_layout_mode");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "layout_mode", PROPERTY_HINT_ENUM, "Position,Anchors,Container,Uncontrolled", (PROPERTY_USAGE_DEFAULT & ~PROPERTY_USAGE_STORAGE) | PROPERTY_USAGE_INTERNAL), "_set_layout_mode", "_get_layout_mode");
```

Comment:
This should likely just be `PROPERTY_USAGE_INTERNAL`.

Reply:
Yes it can be simplified to `PROPERTY_USAGE_EDITOR | PROPERTY_USAGE_INTERNAL` if we don't expect `PROPERTY_USAGE_DEFAULT` to change.